build.sbt

```diff
@@ -78,7 +78,10 @@ lazy val catsInterop = crossProject(JSPlatform, JVMPlatform)
   .settings(name := "caliban-cats")
   .settings(commonSettings)
   .settings(
-    libraryDependencies ++= Seq("org.typelevel" %%% "cats-effect" % "2.0.0")
+    libraryDependencies ++= Seq(
+      "dev.zio"       %% "zio-interop-cats" % "2.0.0.0-RC6",
```

Comment:
I think you need a triple `%%%` for ScalaJS here (I think it only complains about this at runtime).